expression/chunk_executor.go

```diff
@@ -95,8 +95,10 @@ func evalOneVec(ctx sessionctx.Context, expr Expression, input *chunk.Chunk, out
 		}
 		if ft.Tp == mysql.TypeFloat {
 			f64s := result.Float64s()
-			buf := chunk.NewColumn(ft, input.NumRows())
-			buf.ResizeFloat32(input.NumRows())
+			n := input.NumRows()
+			buf := chunk.NewColumn(ft, n)
+			buf.ResizeFloat32(n)
+			buf.SetNulls(0, n, false)
```

Comment:
We may make nulls be false to default after `buf.ResizeXXX`, so that the caller need not this statement.

Reply:
Updated, now after ResizeXXX, all slots are set to not null.

Reply:
After rethinking, it would be better that add an argument in `buf.ResizeXXX` describing `isNull` or not, explicitly.

Reply:
A new argument `isNull` has been added, PTAL~